src/ResourceManagement/TrafficManager/Microsoft.Azure.Management.TrafficManager/generate.cmd

```diff
@@ -4,9 +4,9 @@
 ::
 
 @echo off
-set autoRestVersion=0.14.0-Nightly20160125
+set autoRestVersion=0.16.0
 if  "%1" == "" (
-    set specFile="https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-trafficmanager/2015-11-01/swagger/trafficmanager.json"
+    set specFile="https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-trafficmanager/2017-03-01/swagger/trafficmanager.json"
```

Comment:
Once the rest-api-specs PR is merged, you will need to update this to reference the last commit in that PR, and not the floating reference to 'master'